components/scream/src/share/grid/remap/coarsening_remapper.cpp

```diff
@@ -304,8 +320,10 @@ local_mat_vec (const Field& x, const Field& y) const
   using PackInfo    = ekat::PackInfo<PackSize>;
 
   const auto& src_layout = x.get_header().get_identifier().get_layout();
+  const auto& tgt_layout = y.get_header().get_identifier().get_layout();
```

Comment:
This seems to be unused.

Reply:
oh yes, good catch, these are leftover from some debugging.